### PR TITLE
HTML to ipynb update for scatter and line charts

### PR DESCRIPTION
**Pull request by priyatharsan**
Update for scatter, line charts and line and scatter #802

### CODE REVIEW

File: _posts/matplotlib/line_and_scatter/2015-06-30-line_and_scatter.html [hunk 1]
```diff
@@ -0,0 +1,292 @@
+---
+permalink: matplotlib/line-and-scatter/
+description: How to make line and scatter plots in matplotlib. Seven examples of basic and colored line and scatter plots.
```
Comment (bcdunbar) [inline _posts/matplotlib/line_and_scatter/2015-06-30-line_and_scatter.html]: I'd remove this ~Seven examples of basic and colored line and scatter plots.~

File: _posts/matplotlib/line_and_scatter/2015-06-30-line_and_scatter.html [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+name: Line and Scatter Plots
+has_thumbnail: true
+thumbnail: thumbnail/line-and-scatter.jpg
+layout: user-guide
+language: matplotlib
+title: Matplotlib Line and Scatter Plots | Plotly
+display_as: chart_type
```
Comment (bcdunbar) [inline _posts/matplotlib/line_and_scatter/2015-06-30-line_and_scatter.html]: basic?

**Comment by cldougl** [comment]
>Unless the Matplotlib URLs are like that for searching purposes @cldougl ? Though I don't think it would be.

We probably are trying to get results from 'matplotlib plot' with https://plot.ly/matplotlib/plot/

For now I would suggest keeping  https://plot.ly/matplotlib/plot/ and then consolidating:

https://plot.ly/matplotlib/lines-and-markers/
https://plot.ly/matplotlib/scatter/
https://plot.ly/matplotlib/line-and-scatter/

To:
Scatter Plots https://plot.ly/matplotlib/line-and-scatter/
Line Charts https://plot.ly/matplotlib/line-charts/

and adding redirects for the old urls: 

https://plot.ly/matplotlib/lines-and-markers/ --> https://plot.ly/matplotlib/line-charts/
https://plot.ly/matplotlib/scatter/ --> https://plot.ly/matplotlib/line-and-scatter/

how does that sound @bcdunbar @priyatharsan ?

**Comment by bcdunbar** [comment]
Agreed! sg 👍

**Comment by bcdunbar** [comment]
@priyatharsan sweet 👍  

Could you sweep through the `.ipynb`s and ensure that:

- [ ] imports follow the suggested method (as mentioned in other PR https://github.com/plotly/documentation/pull/1048#discussion_r212608405)
- [ ] capitalize all headings
- [ ] remove commented API auth line (as mentioned in other PR https://github.com/plotly/documentation/pull/1048#discussion_r212922857)

**Pull request merged**